tests/powermon/test_serial_port.py

```diff
@@ -0,0 +1,23 @@
+from unittest import TestCase
+from unittest.mock import patch, Mock
+from powermon.ports.serialport import SerialPort
+
+class test_serial_port(TestCase):
```

Comment:
Can you add a test that demonstrates whether a serial port keeps a persistent connection or reconnects with each call?